### PR TITLE
chore: npx => bunx

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [x] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     --      |
| 🇨🇳 Chinese |     --      |

### CODE REVIEW

File: .github/workflows/preview-build.yml [hunk 1]
```diff
@@ -29,11 +29,13 @@ jobs:
...
       - name: run e2e test
         run: bun run test:site
+      - run: |
+          zip -r site.zip _site
       - name: upload site artifact
         uses: actions/upload-artifact@v4
         with:
           name: site
-          path: _site/
+          path: site.zip
```
Comment (afc163) [inline .github/workflows/preview-build.yml:38]:
```suggestion
          path: _site/
```

File: .github/workflows/preview-deploy.yml [hunk 2]
```diff
@@ -86,15 +86,16 @@ jobs:
           workflow: ${{ github.event.workflow_run.workflow_id }}
           run_id: ${{ github.event.workflow_run.id }}
           name: site
-
+      - run: |
+          unzip site.zip
```
Comment (afc163) [inline .github/workflows/preview-deploy.yml:90]:
```suggestion

```

File: .github/workflows/preview-deploy.yml [hunk 3]
```diff
@@ -86,15 +86,14 @@ jobs:
           workflow: ${{ github.event.workflow_run.workflow_id }}
           run_id: ${{ github.event.workflow_run.id }}
           name: site
```
Comment (afc163) [inline .github/workflows/preview-deploy.yml:88]:
```suggestion
          name: site

```

**Comment by afc163** [comment]
来个 +1

**Review by li-jia-nan — APPROVED** [review]

**Pull request merged**